.github/workflows/main.yml

```diff
@@ -43,19 +46,88 @@ jobs:
         run: yarn validate
       - name: Test
         run: yarn test
-      - name: Export S3 Payloads
+      - name: Pack
+        run: yarn pack
+      - name: Cache packed operations
+        id: cache-packed-operations
+        uses: actions/cache@v3
+        with:
+          path: api3-operations-v0.0.1-d.tgz
+          key: ${{ github.sha }}-pack
+      - name: Export S3 payloads
         run: yarn export-ops-json-payloads
       - name: Set AWS credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
-      - name: Upload JSON Payloads with commit hash to S3
+      - name: Upload JSON payloads with commit hash to S3
         run: aws s3 cp json-exports s3://operations-development/${{ github.sha }} --acl public-read --recursive
-      - name: Upload JSON Payloads as latest to S3
+      - name: Upload JSON payloads as latest to S3
         run: aws s3 sync s3://operations-development/${{ github.sha }} s3://operations-development/latest --acl public-read
 
+  build-explorer:
+    name: Build the API3 Explorer
+    runs-on: ubuntu-latest
+    needs: [lint-build-test-publish]
+    steps:
+      - name: Clone
+        uses: actions/checkout@v3
+        with:
+          repository: 'api3dao/api3-explorer'
+          ref: 'remove-relative-homepage'
+          token: ${{secrets.EXPLORER_PAT}}
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.TARGET_NODE_VERSION }}
+      - name: Install dependencies
+        run: yarn install
+      - name: Retrieve cached packed operations
+        id: cache-packed-operations
+        uses: actions/cache@v3
+        with:
+          path: api3-operations-v0.0.1-d.tgz
+          key: ${{ github.sha }}-pack
+      - name: Unpack operations
+        run: |
```

Comment:
Just wondering what does the `|` do there 🤔

Reply:
Hey, it's the multi-line operator for YAML, so it lets you have an inline stanza across multiple lines.
The symbol varies between implementations of YAML.